### PR TITLE
Added code for language issue

### PR DESCRIPTION
**Pull request by pranav1698**
Added code corresponding to issue #146

### CODE REVIEW

**Comment by praveenojha33** [comment]
@pranav1698 Please provide a demo link.

**Review by yohanym95 — APPROVED** [review]

**Comment by pranav1698** [comment]
hey @praveenojha33,
Actually I am new to open source development so sorry but I don't know how to create a demo link 
If you could help me for the same, I have tested my changes in my localhost

**Comment by pranav1698** [comment]
You can check it now,
https://pranav1698.fwd.wf

**Comment by yohanym95** [comment]
provide a link, not a language.

**Comment by praveenojha33** [comment]
@pranav1698 Please use surge or heroku. Please do a little search on internet for common queries.

**Comment by pranav1698** [comment]
http://c35152c4.ngrok.io/ this is the link for the demo 
I have used local tunneling for the demo link;

**Comment by abishekvashok** [comment]
Link returns 404

**Comment by pranav1698** [comment]
Hey @abishekvashok;
Actually the site was on local tunelling and I could'nt open my workstation yesterday;
You can check it now, the link is: http://c674b602.ngrok.io/

**Comment by pranav1698** [comment]
Were you not able to check in the above link

**Comment by pranav1698** [comment]
No the instructions clearly indicate that you can view your changes on your computer only;
If you could help me when you are free; then I will host the demo link at that time; as when I used surge and heroku the css was not loading
At this moment the demo link is: http://d9e3a17b.ngrok.io/

**Comment by pranav1698** [comment]
No the instructions clearly indicate that you can view your changes on your computer only;
If you could help me when you are free; then I will host the demo link at that time; as when I used surge and heroku the css was not loading
At this moment the demo link is: http://b7a20d99.ngrok.io/

**Comment by abishekvashok** [comment]
If you could provide screenshots it will be okay as well

**Comment by pranav1698** [comment]
Sure,  abhishek on it

**Comment by abishekvashok** [comment]
Maybe along with it provide a gif or video

**Comment by pranav1698** [comment]
Hey @abishekvashok, 
The gif for the language issue
publiccode

**Review by abishekvashok — APPROVED** [review]

**Review by praveenojha33 — APPROVED** [review]

**Review by samagragupta — CHANGES_REQUESTED** [review]

File: site/static/css/creative.css [hunk 1]
```diff
@@ -602,3 +602,28 @@ body {
...
+  
+  display: block;
+  text-align: center
+}
+
+.drop-content a:hover {
+  background-color: rgb(214, 29, 29);
+  color: white;
+  font-weight: bold
+}
```
[the remark this answers is not in the document]
Comment (pranav1698) [reply site/static/css/creative.css]: Sorry for the delay

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by sarnava1 — APPROVED** [review]

**Comment by akash-arelli** [comment]
I would like to work  this issue. The dropdown list shoes languages in short form which is confusing. I would like to add original languages name in original language.

**Pull request merged**